company_website/templates/common_partials/head.haml

```diff
@@ -18,6 +19,10 @@
 %meta{:name => "viewport",
         :content => "width=device-width, initial-scale=1, maximum-scale=5, user-scalable=yes"}
 
+{% if request.get_full_path|is_pagination_suffix %}
+    %meta{:name => "robots", :content => "noindex,follow"}
```

Comment:
Missing space between `noindex` and `follow` :)

Reply:
Fixed